src/components/side-menu/index.ts

```diff
@@ -0,0 +1,4 @@
+export { default as SideMenu } from './SideMenu';
```

Comment:
I'm not sure we should export SideMenu since it is not meant to be used directly

Reply:
solved